content/community/events/Gen3workshop.adoc

```diff
@@ -0,0 +1,274 @@
+---
```

Comment:
please review https://github.com/uc-cdis/gen3.org/pull/164 for asciidoc files to be supported